src/main/java/com/example/bookapi/dto/BookSearchParametersDto.java

```diff
@@ -0,0 +1,4 @@
+package com.example.bookapi.dto;
+
+public record BookSearchParametersDto(String[] author, String[] title) {
```

Comment:
task is:
Add CustomGlobalExceptionHandler class
Add meaningful validation for existing DTO classes

Why this PR contains BookSearchParametersDto?